tsconfig.json

```diff
@@ -3,10 +3,13 @@
         "outDir": "build",
         "module": "esNext",
         "target": "es6",
-        "allowJs": true,
-        "noImplicitAny": false,
+        "allowJs": false,
```

Comment:
Because we don't have JS files... if there's a good reason why we'd add JS files to a 100% typescript project we should take this out. My perspective is not turning this off opens to door to future engineers not conforming to TS (could be wrong)

Reply:
If we're limiting our source to `ts` files, let's remove the notion of `js` files from our webpack configs.

Reply:
I turned this back on, I had issues with a node module importing another file that happened to be JS, not sure if it was a result of this change or a related change of fully replacing babel with typescript. Going to keep JS and babel in until we can figure out a simple way to enforce TS in the source.